pkgs/development/tools/sourcetrail/default.nix

```diff
@@ -194,20 +252,24 @@ stdenv.mkDerivation rec {
     popd
 
     rm ../bin/app/data/{python,java/lib}
+
+    runHook postCheck
   '';
 
   # This has to be done manually in the installPhase because the actual binary
   # lives in $out/opt/sourcetrail/bin, which isn't covered by wrapQtAppsHook
   dontWrapQtApps = true;
 
-  # FIXME: some test cases are disabled in the patch phase
-  doCheck = true;
+  # FIXME: Some test cases are disabled in the patch phase.
+  # FIXME: Tests are disabled on some platforms because of faulty detection
+  # logic for libjvm.so. Should work with manual configuration.
+  doCheck = !stdenv.isDarwin && stdenv.isx86_64;
 
   meta = with lib; {
     homepage = "https://www.sourcetrail.com";
     description = "A cross-platform source explorer for C/C++ and Java";
     platforms = platforms.all;
-    license = licenses.gpl3;
+    license = licenses.gpl3Plus;
```

Comment:
Should the license also be changed in `python.nix`? Please also make a separate commit for license changes in the future.

(I noticed this in my initial review, but then for some reason missed it when I wanted to comment on it.)

Reply:
I missed this, thanks for the tip. I'll include the change in the next update.